tests/pytests/test_info_modules.py

```diff
@@ -505,6 +505,8 @@ def test_redis_info_modules_vecsim():
   info = env.cmd('INFO', 'MODULES')
   field_infos = [to_dict(env.cmd(debug_cmd(), 'VECSIM_INFO', f'idx{i}', 'vec')) for i in range(1, 4)]
   env.assertEqual(info['search_used_memory_vector_index'], sum(field_info['MEMORY'] for field_info in field_infos))
+  # Validate that vector indexes are accounted in the total index memory
+  env.assertGreater(info['search_used_memory_indexes'], info['search_used_memory_vector_index'])
```

Comment:
Consider elaborating the cpp tests to achieve a more accurate comparison
Refer to `test_cpp_llapi.cpp:testInfoSize`

Reply:
also do we have a test that compares the `IndexSpec_TotalMemUsage` increase with vecsim allocator debug info ?